### PR TITLE
Add README contents

### PR DESCRIPTION
**Pull request by casperboone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,129 @@
...
+
+## Credits
+- [Joël Abrahams](https://github.com/jsabrahams)
+- [Georgios Andreadis](https://github.com/gandreadis)
+- [Casper Boone](https://github.com/casperboone)
+- [Felix Dekker](https://github.com/fwdekker)
+- [All Contributors](../../contributors)
+
+## License
+The MIT License (MIT). Please see the [license file](LICENSE.md) for more information.
```
Comment (JSAbrahams) [inline README.md]: Hmmm this link doesn't work

File: README.md [hunk 2]
```diff
@@ -4,3 +4,129 @@
...
+## Changelog
+Please see [releases](../../releases) for more information on what has changed recently.
+
+## Testing
+``` bash
+$ ./gradlew check
+```
+
+## Contributing
+Please see [CONTRIBUTING](CONTRIBUTING.md) for details.
```
Comment (JSAbrahams) [inline README.md]: Hmm this link doesn't work

File: README.md (hunk 1 above)
Comment (casperboone) [reply README.md]: Good catch!

File: README.md (hunk 2 above)
Comment (casperboone) [reply README.md]: See #104

Comment (JSAbrahams) [reply README.md]: Ah nice, however, this link will likely fail as well due to the '.md' extension (similar to `LICENSE`). Have you checked this?

Comment (JSAbrahams) [reply README.md]: Ah alright, then it's fine!

**Pull request merged**